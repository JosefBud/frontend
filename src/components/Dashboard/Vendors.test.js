import React from 'react';
import renderer from 'react-test-renderer';
import { Vendors } from './Vendors';
import { withMemoryRouter } from 'utils';
// Prevent findDOMNode error in test from Dropdown component in react-bootstrap
jest.mock('react-dom', () => ({
  findDOMNode: () => ({})
}));

describe('Dashboard <Vendors />', () => {
  const props = {
    layoutOptions: {
      border: false,
      header: true,
      title: false,
      style: {}
    },
    collection: [],
    pager: {}
  };

  const RoutedVendors = withMemoryRouter(Vendors);

  it('renders correctly', () => {
    expect(
      renderer.create(<RoutedVendors {...props} />).toJSON()
    ).toMatchSnapshot();
  });
});
