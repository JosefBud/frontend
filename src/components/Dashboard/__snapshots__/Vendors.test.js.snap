// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard <Vendors /> renders correctly 1`] = `
<div
  className="card"
  style={Object {}}
>
  <div
    className="card-header"
  >
    Ingredients &gt; Vendors
  </div>
  <div
    className="card-body"
  >
    
    
    <div
      className="table-responsive"
    >
      <table
        className="table table-sm table-striped table-bordered table-hover"
      >
        <thead>
          <tr
            className="text-center"
          >
            <th>
              ID
            </th>
            <th>
              Name
            </th>
            <th>
              Slug
            </th>
            <th>
              Code
            </th>
            <th>
              Options
            </th>
          </tr>
        </thead>
        <tbody />
      </table>
    </div>
    <div
      className="container-fluid"
    >
      <div
        className="row"
      >
        <div
          className="col"
        >
           
          Vendors
        </div>
        <div
          className="text-right col"
        >
           Pages
        </div>
      </div>
    </div>
  </div>
</div>
`;
